aio/scripts/contributors/validate-data.js

```diff
@@ -58,6 +59,32 @@ function _main() {
     }
   }
 
+  // Verify the provided contributor websites are reachable via http(s).
+  for (const [key, entry] of Object.entries(contributors)) {
+    if (entry.website === undefined) {
+      continue;
+    }
+
+    let url;
+    try {
+      url = new URL(entry.website);
+      if (url.protocol !== 'http:' && url.protocol !== 'https:') {
+        throw Error;
+      }
+    } catch {
+      throw Error(`The '${key}' entry links to a website which is not a valid http(s) URL: ${entry.website}`);
+    }
+
+    try {
+      const result = await fetch(url);
+      if (!result.ok && (result.status === 999 && !url.hostname.includes('linkedin.com'))) {
```

Comment:
Could you add a comment on the LinkedIn issue.

Also, the current condition will only be true for `999` responses (on any domain other than `linkedin.com`) :grin:
You probably meant:

```suggestion
      if (!result.ok && !(result.status === 999 && url.hostname.includes('linkedin.com'))) {
```

...or...

```suggestion
      if (!result.ok && (result.status !== 999 || !url.hostname.includes('linkedin.com'))) {
```

Reply:
Updated the comments to make it more clear.